app/src/popup/Popup.tsx

```diff
@@ -17,13 +18,19 @@ export const Popup = (): JSX.Element => {
 
     return (
         <div className="popup-view">
-            {expanded ? <ExpandedView setSortingOrder={setSortingOrder} sorted={order} /> : <ShrinkedView />}
-            <Button
-                label={expanded ? "shrink-button-label" : "expand-button-label"}
-                onClick={() => {
-                    setExpanded(!expanded);
-                }}
-            />
+            {expanded ? (

```

Comment:
Initially this was the idea of handling a navigation in SPA-like add-on, however, currently I start to recognize some major problems with it.
For now, let it stay as it is, but this part of code should utilize the react-router at least.